packages/client/tests/functional/issues/9007/tests.ts

```diff
@@ -24,17 +22,7 @@ testMatrix.setupTestSuite(
       ).rejects.toThrowError()
     })
 
-    test('should not generate the contains field on the where type', async () => {
-      const generatedTypeScriptSrc = await fs.promises.readFile(
-        path.join(testRoot, 'node_modules', '@prisma', 'client', 'index.d.ts'),
-        'utf-8',
-      )
-
-      const [uuidFilter] = generatedTypeScriptSrc.split('export type UuidFilter = {')[1].split('}')
-
-      const hasContains = uuidFilter.includes('contains')
-
-      expect(hasContains).toEqual(false)
+    test('should not generate the contains field on the where type', () => {
       expectTypeOf<Prisma.UuidFilter>().not.toHaveProperty('contains')
```

Comment:
This check does exactly the same but on the type-level, so not reading the file, but using the type system. That's why I am removing the code above.